_posts/2024-03-13-java-22.md

```diff
@@ -50,7 +50,7 @@ community:
 * Scoped Values in second preview  ([JEP 464](https://openjdk.org/jeps/464))
 
 Many of these API's allow us to further modernize the feature set of Trino and
-adapt it to current hardware and compute power realtities. Specifically we can
+adapt it to current hardware and compute power realties. Specifically we can
```

Comment:
It’s supposed to be “realities”

Reply:
I accidentally removed `i`. Sent another PR: https://github.com/trinodb/trino.io/pull/642